sample/src/Methods.tsx

```diff
@@ -21,7 +21,7 @@ export default function Methods() {
       <MethodCall
         name="initialize"
         call={() => {
-          Didomi.initialize(apiKey).catch((err) => {
+          Didomi.initializeWithParameters({ apiKey: apiKey, noticeId: "mBcaPUEX", androidTvEnabled: true, androidTvNoticeId: "mBcaPUEX" }).catch((err) => {
```

Comment:
`mBcaPUEX` is a TV notice id, for mobile we should put the same notice that we use in tests.

Also this notice was not created specifically for our sample app, it's just one of the random notices we use for CTV demos. But as long as we don't rely on it for automated tests, it should be ok.